defactor-com/ui-kit
# Ui Kit Improved

## Create component

### What does this PR do?

- Resolve #225
- Resolve #228 

### Steps to test

1. Run `yarn storybook`
2. Check changes

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/7525bcaa-05cf-4268-956d-b869ad105922)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/0ed2aa3c-ce9d-4a1a-a727-374b3e336365)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/9b39a2dc-b494-42dd-92e5-8f9ed4d70726)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/92767178-676d-425a-a739-0ab85d4f2ef9)


#### CheckList
